src/components/layout/inputs/CheckboxInput/CheckboxInput.jsx

```diff
@@ -59,6 +65,7 @@ CheckboxInput.propTypes = {
   checked: PropTypes.bool.isRequired,
   className: PropTypes.string,
   hiddenLabel: PropTypes.bool,
+  isError: PropTypes.bool,
```

Comment:
```suggestion
  inError: PropTypes.bool,
```

Je trouve que ça a plus de sens non ?
Ou `isInError` ?

Reply:
Aucune préférence, les deux sont équivalent pour moi.